src/api/nodeRpcCall.js

```diff
@@ -1326,6 +1326,35 @@ const getStakersRewards = async (accounts) => {
   return await api.derive.staking.stakerRewardsMulti(accounts, false);
 }
 
+const getSessionValidators = async () => {
+  const api = await getApi();
+  const rawData = await api.query.session.validators();
+  return rawData.map(v => v.toString());
+}
+
+const getNextSessionValidators = async () => {
+  const api = await getApi();
+  const data = await api.query.session.queuedKeys();
+  return data.map(([validator]) => validator.toString());
+}
+
+const getStakingValidators = async () => {
+  const api = await getApi();
+  const rawData = await api.query.staking.validators.keys()
+  return rawData.map(v => v.args[0].toString());
+}
+
+const getNominators = async () => {
+  const api = await getApi();
+  const rawData = await api.query.staking.nominators.keys()
+  return rawData.map(v => v.args[0].toString());
+}
+
+const getStakingLedger = async (controller) => {
```

Comment:
what is this ?

Reply:
`staking.ledger` is a map from controller account id to details on staking, which include:
* stash that's controlled by this controller
* staked amount
* if account is onlocking some stake
* list of eras for which this controller already claimed the staking rewards

We use it to:
* get the stash account, in case citizen's account is a controller but not stash (this PR)
* decide for which eras we still need to do a payout for the payout all button

![image](https://github.com/liberland/liberland_frontend/assets/117277751/9dc9b9d5-21aa-406c-bdd8-36207e7fb134)